src/test/java/com/puppycrawl/tools/checkstyle/checks/imports/AvoidStaticImportCheckTest.java

```diff
@@ -50,17 +50,16 @@ public void testDefaultOperation()
         final DefaultConfiguration checkConfig =
             createModuleConfig(AvoidStaticImportCheck.class);
         final String[] expected = {
-            "23: " + getCheckMessage(MSG_KEY, "java.io.File.listRoots"),
-            "25: " + getCheckMessage(MSG_KEY, "javax.swing.WindowConstants.*"),
-            "26: " + getCheckMessage(MSG_KEY, "javax.swing.WindowConstants.*"),
-            "27: " + getCheckMessage(MSG_KEY, "java.io.File.createTempFile"),
-            "28: " + getCheckMessage(MSG_KEY, "java.io.File.pathSeparator"),
-            "29: " + getCheckMessage(MSG_KEY, "java.lang.Math.E"),
-            "30: " + getCheckMessage(MSG_KEY, "java.lang.Math.sqrt"),
-            "31: " + getCheckMessage(MSG_KEY,
+            "23:27: " + getCheckMessage(MSG_KEY, "java.io.File.listRoots"),
+            "25:42: " + getCheckMessage(MSG_KEY, "javax.swing.WindowConstants.*"),
+            "26:27: " + getCheckMessage(MSG_KEY, "java.io.File.createTempFile"),
+            "27:27: " + getCheckMessage(MSG_KEY, "java.io.File.pathSeparator"),
+            "28:29: " + getCheckMessage(MSG_KEY, "java.lang.Math.E"),
+            "29:29: " + getCheckMessage(MSG_KEY, "java.lang.Math.sqrt"),
+            "30:113: " + getCheckMessage(MSG_KEY,
                 "com.puppycrawl.tools.checkstyle.checks.imports."
                     + "avoidstaticimport.InputAvoidStaticImportNestedClass.InnerClass"),
-            "32: " + getCheckMessage(MSG_KEY,
+            "31:124: " + getCheckMessage(MSG_KEY,
```

Comment:
Please confirm why line number changed from 32 to 31. 
https://github.com/checkstyle/checkstyle/blob/555199e6bbb338ef84afffd1a288100041f0aae9/src/test/resources/com/puppycrawl/tools/checkstyle/checks/imports/avoidstaticimport/InputAvoidStaticImportDefault.java#L31
I am not seeing a reason why in the file as it doesn't span more than 1 line.

Reply:
I was asked by @strkkk to remove the input duplicate import

Reply:
Shall I revert?

Reply:
https://github.com/checkstyle/checkstyle/pull/476
I don't see any reason for the duplicate import. Normally we wouldn't change inputs like this as they may be testing something. Additionally, to remove a line from a file with violations will cause the entire test's line numbers to change.

If you want to keep this change, it must be another commit since it is outside the scope of this issue.

Reply:
Okay, so it should be a different PR altogether or 2 commits in this PR?

Reply:
second commit with message like "minor: remove duplicated violation from AvoidStaticImport"

Reply:
Sure